app/src/main/java/com/orange/ouds/app/ui/tokens/TokenCategory.kt

```diff
@@ -87,24 +86,18 @@ sealed class TokenCategory<T>(
                 TokenProperty.SpaceScaled,
                 TokenProperty.SpaceFixed,
                 TokenProperty.SpacePaddingInline,
-                TokenProperty.SpacePaddingInlineWithIcon,
-                TokenProperty.SpacePaddingInlineWithArrow,
                 TokenProperty.SpacePaddingStack,
-                TokenProperty.SpacePaddingStackWithIcon,
                 TokenProperty.SpacePaddingInset,
                 TokenProperty.SpaceColumnGap,
-                TokenProperty.SpaceColumnGapWithIcon,
-                TokenProperty.SpaceColumnGapWithArrow,
                 TokenProperty.SpaceRowGap,
-                TokenProperty.SpaceRowGapWithIcon
             ),
         )
 
         data object Size : TokenCategory<Size>(
             R.string.app_tokens_dimension_size_label,
             R.drawable.ic_dimension,
             R.string.app_tokens_dimension_size_description_text,
-            getTokenValueCode<OudsSizeKeyToken.Icon.Decorative.Shortest>(),
+            getTokenValueCode<OudsSizeKeyToken.Icon.Decorative.Small>(),
```

Comment:
A detail: code example usually use the first value, which would be `ExtraExtraSmall` here.